src/aero_model.F90

```diff
@@ -598,11 +603,16 @@ subroutine aero_model_emissions( state, cam_in )
     end if
 
     ! Pick up correct DMS emissions (replace values from file if requested)
-    ! If DMS is sent from the ocean then cflx is updated in the nuopc cap
     if (.not. dms_from_ocn) then
        call oslo_aero_dms_emis(state%ncol, state%lchnk, &
             state%u(:,pver), state%v(:,pver), state%zm(:,pver), &
             cam_in%ocnfrac, cam_in%icefrac, cam_in%sst, cam_in%cflx)
+    else
+       call cnst_get_ind('DMS', pndx_fdms, abort=.true.)
+       do icol = 1,state%ncol
+          cam_in%cflx(icol,pndx_fdms) = cam_in%fdms(icol)
+          call outfld('odms', cam_in%fdms(:state%ncol), state%ncol, state%lchnk)
```

Comment:
Can you document that the `addfld` for this is in oslo_aero_ocean.F90

Reply:
Done.